src/components/structures/auth/SoftLogout.js

```diff
@@ -48,7 +63,21 @@ export default class SoftLogout extends React.Component {
             domainName,
             userId,
             displayName,
+            loginView: LOGIN_VIEW.LOADING,
+            keyBackupNeeded: true, // assume we do while we figure it out (see componentWillMount)
+
+            busy: false,
+            password: "",
+            errorText: "",
         };
+
+        this._initLogin();
+    }
+
+    componentWillMount(): void {
+        MatrixClientPeg.get().flagAllGroupSessionsForBackup().then(remaining => {
```

Comment:
recommended to do async stuff in componentDidMount

Reply:
Also, as mentioned before, `flagAllGroupSessionsForBackup` will mark the store as needing a backup regardless of `remaining`... I suppose that won't have any undesirable consequences?

Reply:
It shouldn't have consequences (at worst it backs up your keys when you log in again). We also specifically don't start the backup to avoid the undesirable consequences